db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/user_remove_vm_finished.yaml

```diff
@@ -9,8 +9,8 @@ object:
     description: 
   fields:
   - rel4:
-      value: "/System/event_handlers/src_vm_disconnect_storage"
-  - rel5:
       value: "/System/event_handlers/event_action_policy?target=src_vm&policy_event=vm_unregister&param="
+  - rel5:
+      value: "/System/event_handlers/event_action_refresh_sync?target=src_vm"
```

Comment:
right, but even before it was accessing storage and storages relation, which depends on refresh.

https://github.com/Ladas/manageiq/blob/b0004a7b3f573f723ea8152f128175fb57fef2bd/app/models/vm_or_template.rb#L724-L724